tools/oprt2/pkg/packagemanager/apt/config.go

```diff
@@ -0,0 +1,73 @@
+/*
+ *  Copyright 2025 Gravitational, Inc
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package apt
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+	"regexp"
+
+	"github.com/gravitational/shared-workflows/tools/oprt2/pkg/attunehooks/gpg"
+	"github.com/gravitational/shared-workflows/tools/oprt2/pkg/commandrunner"
+	"github.com/gravitational/shared-workflows/tools/oprt2/pkg/config"
+	"github.com/gravitational/shared-workflows/tools/oprt2/pkg/filemanager"
+)
+
+// FromConfig creates a new APT instance from the provided config, and optional Attune hooks.
+func FromConfig(ctx context.Context, config config.APT, logger *slog.Logger, attuneHooks ...commandrunner.Hook) (*APT, error) {
```

Comment:
Same naming nit as before. To me it seems like this could just be named `New` so you'd use it as `apt.New(...)`.